OutSystems/outsystems-datagrid
# ROU-4181: Action Column Revamp

This PR is allow the action column to display buttons in addition to the links.

### What was done
* Created a new ColumnConfigAction class to add the actionColumnElementType property
* Created the ActionColumnType enum to enumerate the element type
* Created the ActionColumnFactory that returns the cellTemplate based on the Action Column's Element Type
* Updated the ActionColumns class to accept the new config
* Updated the Grid.css file to add the Action Column Button style.

### Test Steps
1. In a Grid, add a Action Column
2. Set the ElementType parameter to Button
3. Add a Click event handler
4. Publish the module
5. Check if the column works as expected.

### Checklist
